File: playframework/app/models/Book.scala
package models

import play.api.libs.json.Json
import com.github.nscala_time.time.Imports._
import spray.json._

case class Book(title: String, authors: Seq[String], publisher: String, isbn13: Long, publishedDate: String)
//{
//	val formatedDate: String = DateTimeFormat.forPattern("MMMM d, yyyy").print(publishedDate)
//}

object MyJsonProtocol extends DefaultJsonProtocol {
  implicit val bookFormat = jsonFormat5(Book.apply)
}

object Book {
  implicit val bookFmt = Json.format[Book]
  private var books = 
    Array(
        Book("Animal Farm", Seq("George Orwell"), "Penguin", 9780141036137L, DateTimeFormat.forPattern("MMMM d, yyyy").print(DateTime.now)),
        Book("1984", Seq("George Orwell"), "Penguin", 9780141036144L, DateTimeFormat.forPattern("MMMM d, yyyy").print(DateTime.now)),
        Book("The Razor's Edge", Seq("William Sommerset Maugham"), "Vintage", 9780099284864L, DateTimeFormat.forPattern("MMMM d, yyyy").print(DateTime.now))
    )
    
    def all = books 
}

File: restexpress/src/main/java/org/restexpress/scaffold/minimal/Book.java
package org.restexpress.scaffold.minimal;

/**
 * Created by alexandrejbr on 07/05/14.
 */
public class Book
{
    public static Book[] all =
            new Book[] {
                new Book("Animal Farm", new String[]{"George Orwell"}, "Penguin", 9780141036137L, "April 27, 2014"),
                new Book("Animal Farm", new String[]{"George Orwell"}, "Penguin", 9780141036137L, "April 27, 2014"),
                new Book("Animal Farm", new String[]{"George Orwell"}, "Penguin", 9780141036137L, "April 27, 2014"),
            };

    public String title;
    public String[] authors;
    public String publisher;
    public long isbn13;
    public String publishedDate;

    public Book(String title, String[] authors, String publisher, long isbn13, String publishedDate) {
        this.title = title;
        this.authors = authors;
        this.publisher = publisher;
        this.isbn13 = isbn13;
        this.publishedDate = publishedDate;
    }
}
